src/main/resources/db/changelog/changes/001-initial-schema.sql

```diff
@@ -0,0 +1,58 @@
+--liquibase formatted sql
+--changeset wfeliciano20:create_movies_table-001
```

Comment:
The second part of this line must be a number, a sequence: `wfeliciano20:1`. You don't need a changeset per SQL query if you want to group them all togheter.